### PR TITLE
[miio] add support for zhimi.airpurifier.vb2 Mi Air Purifier Pro H

### LINKED ISSUE #10152: [miio] Please support Qingping Air Monitor Lite (cgllc.airm.cgdn1)
@marcelrv Please support Qingping Air Monitor Lite (cgllc.airm.cgdn1)

See Miot specs: https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:air-monitor:0000A008:cgllc-cgdn1:1

And the integration in python-miio for reference
https://github.com/rytilahti/python-miio/commit/78f8beee13d9fbd5f2b1a11a311f736b41a2ba06

### LINKED ISSUE #10174: [miio] Add support for Mi Airpurifier Pro H (zhimi.airpurifier.vb2
Hi,
As discussed in the community forums, here is the dump of humidifier data in order to add support for it in miio binding.
[test-zhimi.airpurifier.vb2-20210215-174145.txt](https://github.com/openhab/openhab-addons/files/5989319/test-zhimi.airpurifier.vb2-20210215-174145.txt)

Feel free to buzz me if you need me to test anything related to that before the actual PR.

Cheers,
Konstantin

### PR DESCRIPTION
**Pull request by marcelrv**
close #10174

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

**Comment by marcelrv** [comment]
@theater updated the modes to the ones visible in the app

**Comment by theater** [comment]
Hi Marcel,
Sorry for the confusion. I played a bit with the Mode and it seems that I have provided you false information for the Mode.

0, 1 are OK. 
2 Should be "Favourite" because when enabled it automatically turns the Fan level to the one set in the fan_level channel.
3 should be "Manual" maybe because when turned on in the UI it provides you a slider which you can define the mode based on room square feet.

All in all the settings work and look great.

Thank you very much for adding this to the DB!

**Review by theater — APPROVED** [review]

File: bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.airpurifier.vb2-miot.json [hunk 1]
```diff
@@ -0,0 +1,1098 @@
...
+					}
+				]
+			},
+			{
+				"property": "use-time",
+				"siid": 12,
+				"piid": 1,
+				"friendlyName": "Use Time - Use Time",
+				"channel": "use_time",
+				"type": "Number:Duration",
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.airpurifier.vb2-miot.json:640]:
```suggestion
				"type": "Number:Time",
```
Please update elsewhere as appropriate.

**Review by cpmeister — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.airpurifier.vb2-miot.json (hunk 1 above)
Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.airpurifier.vb2-miot.json]: Yep, this one also had it

**Review by cpmeister — APPROVED** [review]

**Pull request merged**